.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+The following fields can give you some guidance on formatting your
+pull request. Not all fields are relevant for all pulls. Please
+fill in fields that you think are useful; brevity and clarity are
+valued.
+
+-----
+
+### About
+
+Briefly describe what you did, if you need to include technical details, start here.
+
+### Issues
+
+Fixes #012
+
+### Reason
+
+If this change doesn't fix an issue, it would be good to give some background as to why this change is necessary.
+
+### Release Notes
+
+Include possible release notes/documentation, if applicable.
```

Comment:
We aren't using our CHANGELOG file, is it worth it? It certainly would have saved me a little while last Saturday. The biggest problem is we don't have any info for 2.5.8 and down.